### PR TITLE
Replace stanford_theme_enabled with is_non_microsite_theme

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 Please review.

File: lms/templates/main.html [hunk 1]
```diff
@@ -31,14 +31,8 @@
...
-
-<%def name="stanford_theme_enabled()">
-  <%
-    if not theme_enabled():
-      return False
-
-    return getattr(settings, "THEME_NAME", None) == "stanford"
-  %>
+<%def name="is_non_microsite_theme()">
+  <% return theme_enabled() and is_microsite() %>
```
Comment (caesar2164) [inline lms/templates/main.html]: this should be: `theme_enabled() and not is_microsite()`

Comment (stvstnfrd) [inline lms/templates/main.html]: Good catch, change made.

**Comment by caesar2164** [comment]
pending tests, i'm :+1: :wink:

**Comment by stvstnfrd** [comment]
retest this please

**Comment by stvstnfrd** [comment]
cherry-picked.

**Pull request closed without merging**